src/parser.rs

```diff
@@ -415,7 +420,8 @@ rule scenario() -> Scenario
 rule tag_char() -> &'input str
     = s:$([_]) {?
         let x = s.chars().next().unwrap();
-        if x.is_alphanumeric() || x == '_' || x == '-' {
+        // `)` isn't allowed, as it would collide with TagExpression.
```

Comment:
As far as I see [in reference implemetation](https://github.com/cucumber/common/blob/gherkin/v22.0.0/gherkin/go/matcher.go#L111), we should support it.

Maybe this should be resolved by escaping in `TagExpression`?

Reply:
@tyranron somewhere buried in separate github repo, there is mentioning of escaping https://github.com/cucumber/tag-expressions#escaping